docker.go

```diff
@@ -211,10 +211,12 @@ func commandBuild(build Build) *exec.Cmd {
 	args := []string{
 		"build",
 		"--rm=true",
-		"-f", build.Dockerfile,
-		"-t", build.Name,
 	}
 
+	if build.Dockerfile != "Dockerfile" {
+		args = append(args, "-f", build.Dockerfile)
+	}
+	args = append(args, "-t", build.Name)
```

Comment:
Why did you moved this down? This could simply stay in the above definition, ordering of this flag doesn't matter.

Reply:
Agreed, moved 'tag' back